src/core/cells.cpp

```diff
@@ -160,6 +166,24 @@ std::vector<int> mpi_get_short_range_neighbors(int const pid,
                   mpi_get_short_range_neighbors_local, pid, distance, false);
 }
 
+std::vector<Particle *> get_particle_neighbors_local(Particle const &p) {
```

Comment:
```suggestion
static auto get_particle_neighbors(Particle const &p) {
```

* `static`: this function doesn't need to be visible outside of this source file, so marking it as `static` hides it and allows the compiler to inline it
* `auto`: the return type can be deduced since there is only one `return` statement
* `_local`: no longer needed, we have removed almost all `mpi_*_local` functions at this point, developers should now assume that core functions run in parallel by default
* the corresponding prototype can be removed from `cells.hpp`